packages/har-examples/src/index.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable object-shorthand -- Objects that contain shorthand properties can't be inferred with --isolatedDeclarations. */
+import type { Har } from 'har-format';
+
+import application_form_encoded from './application-form-encoded.har.js';
+import application_json from './application-json.har.js';
+import application_zip from './application-zip.har.js';
+import cookies from './cookies.har.js';
+import full from './full.har.js';
+import headers from './headers.har.js';
+import https from './https.har.js';
+import image_png_no_filename from './image-png-no-filename.har.js';
+import image_png from './image-png.har.js';
+import jsonObj_multiline from './jsonObj-multiline.har.js';
+import jsonObj_null_value from './jsonObj-null-value.har.js';
+import multipart_data_dataurl from './multipart-data-dataurl.har.js';
+import multipart_data from './multipart-data.har.js';
+import multipart_file from './multipart-file.har.js';
+import multipart_form_data from './multipart-form-data.har.js';
+import query_encoded from './query-encoded.har.js';
+import query from './query.har.js';
+import short from './short.har.js';
+import text_plain from './text-plain.har.js';
+import xml from './xml.har.js';
+
+type Examples = Record<
+  | 'application-form-encoded'
+  | 'application-json'
+  | 'application-zip'
+  | 'cookies'
+  | 'full'
+  | 'headers'
+  | 'https'
+  | 'image-png-no-filename'
+  | 'image-png'
+  | 'jsonObj-multiline'
+  | 'jsonObj-null-value'
+  | 'multipart-data-dataurl'
+  | 'multipart-data'
+  | 'multipart-file'
+  | 'multipart-form-data'
+  | 'query-encoded'
+  | 'query'
+  | 'short'
+  | 'text-plain'
+  | 'xml',
+  Har
+>;
+
+const examples: Examples = {
```

Comment:
This `Examples` type I had to do is nuts but `isolatedDeclarations` loses its marbles without it:

![Screenshot 2025-03-05 at 6 29 11 PM](https://github.com/user-attachments/assets/d7900aa3-05dc-4548-a4b8-ff0def5c7a3e)

Reply:
Alternatively, if I do ` satisfies Record<string, Har> as Record<string, Har>` like it suggests then the export loses all type inference of what is actually in the `examples` object. Feels like a lose-lose.

Reply:
not sure if i _like like_ that, but i like that.

Reply:
yeahhh the ergonomics of `isolatedDeclarations` is still pretty rough atm. hopefully it's worth it 🫠